pkg/trcapi/conn.go

```diff
@@ -96,14 +97,17 @@ func Connect(ver semver.Version, w io.Writer, r io.Reader) (*Conn, error) {
 
 	resp := &api.Handshake{
 		Version: hs.Version,
+		Token:   hs.Token,
```

Comment:
The token should be decoded from the message, which arrives *from* TRC. We don't send the token to the TRC.